phys2bids/phys2bids.py

```diff
@@ -296,6 +297,12 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
                       phys_in.num_timepoints_found, uniq_freq,
                       phys_out[uniq_freq].start_time, outfile)
 
+    # Generate participants.tsv file if it doesn't exist already.
+    # Update the file if the subject is not in the file.
+    # Do not update if the subject is already in the file.
+    if heur_file:
+        participants_file(indir, outdir, yml, sub)
```

Comment:
In order to save lines of code, this should go in line 282, we avoid the extra if and make sure that sub is given by the user

Reply:
The problem there is we would be calling the function many times inside the for loop. But yeah, I'll add the `sub` check.

Reply:
you could call it in line 267

Reply:
Didn't see this comment. Sure, I'll move it up there.

Reply:
Okay, last commit took care of this. Resolving this comment now.